instance/models/instance.py

```diff
@@ -426,10 +426,14 @@ def provision(self):
         server = self.server_set.create()
         server.start()
 
+        def can_proceed():
```

Comment:
@e-kolpakov I went for just `accepts_ssh_commands`, since it reads a bit better when it's being used below: `server.sleep_until(accepts_ssh_commands)` vs. `server.sleep_until(server_accepts_ssh_commands)`.